src/java/org/apache/cassandra/index/internal/CassandraIndex.java

```diff
@@ -689,33 +705,42 @@ private boolean isPrimaryKeyIndex()
     @SuppressWarnings("resource")
     private void buildBlocking()
     {
-        baseCfs.forceBlockingFlush();
-
-        try (ColumnFamilyStore.RefViewFragment viewFragment = baseCfs.selectAndReference(View.selectFunction(SSTableSet.CANONICAL));
-             Refs<SSTableReader> sstables = viewFragment.refs)
+        try
```

Comment:
All these changes are just a try-catch sandwiching to change `supportedLoads` to NONE on exceptions.